perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -102,29 +103,34 @@ sub construct_waste_open311_update {
 
     $worksheet = $self->whitespace->GetFullWorksheetDetails($worksheet->{id});
 
+    my ($service_id, $config_key);
+    if ($report->category eq 'Report missed collection') {
+        $service_id = MISSED_COLLECTION_SERVICE_PROPERTY_ID;
+        $config_key = 'missed_collection_state_mapping';
+    } else {
+        $service_id = REQUEST_SERVICE_PROPERTY_ID;
+        if ($report->category eq 'Request new container') {
+            $config_key = 'container_deliver_state_mapping';
+        } elsif ($report->category eq 'Request container removal') {
+            $config_key = 'container_collect_state_mapping';
```

Comment:
Wondering again if _collect_ could be _removal_ for (internal) consistency, but I'm probably splitting hairs at this point. 'container_collect' is less ambiguous than 'collect' on its own, so...